src/Components/getList.js

```diff
@@ -27,16 +27,67 @@ function EmptyList() {
   );
 }
 
-function FullList(props) {
+
+const FullList = (props)=> {
   const [isChecked, setCheck] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleChange = e => {
+    setSearchTerm(e.target.value);
+  };
+
+  const clearSearch = () => {
+    setSearchTerm("");
+  };
+
+  const searchList = props.data.filter(item => {
+    return item.itemName.toLowerCase().includes(searchTerm.toLowerCase());
+  });
+
 
 
   return (
     <div className="grocery-list">
       <h1>Groceries</h1>
-      <ul className="tl f2">
-        {props.data.map(item => (
+      <ul className="tl f2 check-list">
+      <li className="bg-white pa1 shadow">
```

Comment:
I encourage you to use correct spacing when typing out HTML, like so:
```html
<ul className="tl f2 check-list">
  <li className="bg-white pa1 shadow">
</ul>
```

All of the formatting can be automated by running `prettier`, which is conveniently installed already in our repo:

```sh
prettier src/Components/getList.js --write
```
This will automatically format the file for you. I tend to configure my code editor to automatically format files via a [code formatter extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) when I save a file. Check it out! https://cdn.zappy.app/d3b7f27daddc258bebc9317dd5d5f107.gif

Reply:
Thank you! Making this easier to read is always good input.

Reply:
😮 I swear we use prettier on a commit hook! This is worth looking at 👀.

Reply:
In a rush to get this done, I neglected formatting.  I thought I had prettier installed in vscode but it looks like I needed to update npm so this should all be configured now, thank you.